### PR TITLE
feat: add a package for Svelte components library

### PR DESCRIPTION
**Pull request by lauthieb**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

Add a Svelte components library.

## Does this introduce a breaking change?

- No

❤️ Thank you!

### CODE REVIEW

File: packages/sources/svelte/.eslintrc.js
Comment (sidbentifraouineatdecathlon) [inline packages/sources/svelte/.eslintrc.js:2]: What do you use for code formatting ? look line the indentation is double space compared to the React project

File: packages/sources/svelte/src/components/index.js
Comment (sidbentifraouineatdecathlon) [inline packages/sources/svelte/src/components/index.js:3]: Just a little suggestion `export { default as Button } from './Button.svelte';`
w/ babel 7 
Proposal: https://github.com/tc39/proposal-export-default-from#common-concerns

**Review by sidbentifraouineatdecathlon — APPROVED** [review]

File: packages/sources/svelte/src/components/index.js
Comment (lauthieb) [reply packages/sources/svelte/src/components/index.js:3]: nice thx, done!

**Review by sidbentifraouineatdecathlon — APPROVED** [review]

File: packages/sources/svelte/preprocess.js
Comment (thollander) [inline packages/sources/svelte/preprocess.js:1]: This script seems to me a bit too "manual", could we consider moving to a more "automated" way ? 
I would recommand using rollup to manage all of this. This is pretty convenient for libraries. 
Also, there is a [component-template](https://github.com/sveltejs/component-template) proprosed by Rich Harris and maintained by Svelte core team. 

If you don't mind, I'd like switching closer to this template before merging this PR. I'll have a look on it. 

Any insights on this ?

Comment (lauthieb) [reply packages/sources/svelte/preprocess.js:1]: Yes feel free Terence! Moreover if we can follow practices from Rich Harris before merging, for sure it would be better!

Comment (lauthieb) [reply packages/sources/svelte/preprocess.js:1]: Is component-template ready for a package `@vtmn/svelte` that will contain several components? Like we did with existing packages for web components & React components.

Comment (thollander) [reply packages/sources/svelte/preprocess.js:1]: It's not ready if we take it "as it is", but it can surely be adapted in the context !

Comment (thollander) [reply packages/sources/svelte/preprocess.js]: Done in https://github.com/Decathlon/vitamin-web/pull/97/commits/2bb5c733553771d8708ade8ad4afc5be42312c53 ✅

File: .husky/.gitignore
Comment (thollander) [inline .husky/.gitignore:1]: Is this `_` wanted ?

File: packages/showcases/svelte/.storybook/preview.js
Comment (thollander) [inline packages/showcases/svelte/.storybook/preview.js:1]:
```suggestion
```

**Review by thollander — APPROVED** [review]
Lgtm 👍🏻

File: .husky/.gitignore
Comment (lauthieb) [reply .husky/.gitignore:1]: I don't know why Husky always create this file on precommit :(

**Comment by lauthieb** [comment]
Thanks @thollander for your commits!!

File: .husky/.gitignore
Comment (thollander) [reply .husky/.gitignore:1]: OK, just wanted to make sure it wasn't a typo!

**Review by gabbloquet — APPROVED** [review]
Lgtm :)👌🏻

File: packages/showcases/svelte/src/stories/components/ButtonView.svelte
Comment (lauthieb) [reply packages/showcases/svelte/src/stories/components/ButtonView.svelte:4]: I've not succeed to write it directly in `Button.stories.js`. I was forced to do this to make it work :/

Comment (thollander) [reply packages/showcases/svelte/src/stories/components/ButtonView.svelte:4]: Yes it's not possible, that's the good way to do it. (I think that it's even explained in storybook docs if I remember well)

**Comment by lauthieb** [comment]
@thollander can you please `git push -f` your local branch?

**Comment by thollander** [comment]
> @thollander can you please `git push -f` your local branch?

Done. ✅ 
There is a conflict, I manage it

**Comment by gabbloquet** [comment]
Thanks @thollander , sorry about that 🙁

**Comment by thollander** [comment]
> Thanks @thollander , sorry about that 🙁

Np! Good now. Let's merge? 👊

**Pull request merged**
